node/src/components/event_stream_server.rs

```diff
@@ -0,0 +1,108 @@
+//! API server
```

Comment:
I think these docs are out of date now that this file is `event_stream_server.rs`

Reply:
Did this.